geocoding/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.0
+
+- Upgrade the plugin to use the federated architecture
```

Comment:
We are trying to keep the CHANGELOG entries inline with the [Google Flutter style guide lines](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog). Which in this case means:

- Entries should use present tense indicative for verbs, with "this version" as an implied subject. For example, "Adds cool new feature.", not "Add" or "Added".
- Entries should end with a `.`.

So in this case a better entry would be:
```suggestion
- Upgrades the app-facing package to use the federated architecture.
```